ocean_lib/data_provider/data_service_provider.py

```diff
@@ -80,7 +81,7 @@ def sign_message(wallet, msg, config, nonce=None):
 
     @staticmethod
     def get_nonce(user_address, config):
-        url = DataServiceProvider.build_endpoint('nonce')
+        method, url = DataServiceProvider.build_endpoint('nonce')
```

Comment:
are you using the `method`? If not, you can replace the variable name with `_` to avoid an unused variable. I am unsure about the business logic here, but maybe you need to check that the nonce endpoint supports GET (with assert, maybe?)

CC: @ssallam any pointers here?